src/ui/public/notify/notify.js

```diff
@@ -18,9 +17,10 @@ module.factory('Notifier', function () {
   return Notifier;
 });
 
-module.run(function ($timeout) {
-  // provide alternate methods for setting timeouts, which will properly trigger digest cycles
-  Notifier.setTimerFns($timeout, $timeout.cancel);
+module.run(function ($interval) {
+  // provide alternate methods for setting intervals
+  // which will properly trigger digest cycles
+  Notifier.setTimerFns($interval, $interval.cancel);
```

Comment:
Initially we were using a timeout to clear the notification after the timeout had elapsed, but now we need to update the view with the seconds remaining.